pkg/util/filehelper/file.go

```diff
@@ -152,3 +152,24 @@ func CopyDir(src string, dst string) (err error) {
 
 	return
 }
+
+// IsDirEmpty checks whether directory is empty or not
+func IsEmptyDir(path string) bool {
+	if !Exists(path) {
+		return true
```

Comment:
I don't think we can do this. If the path does not exist, it really should return _false_.